### PR TITLE
Block deletion of resources with unprocessed events

### PR DESCRIPTION
**Pull request by AlanCoding**

### CODE REVIEW

**Comment by AlanCoding** [comment]
Connected to https://github.com/ansible/awx/pull/1586

Expands upon https://github.com/ansible/awx/pull/1536 and https://github.com/ansible/awx/pull/1533

File: awx/main/constants.py [hunk 1]
```diff
@@ -5,9 +5,17 @@
 
 from django.utils.translation import ugettext_lazy as _
 
+__all__ = [
```
Comment (wwitzel3) [inline awx/main/constants.py:8]: 🎉

File: awx/main/models/ha.py [hunk 2]
```diff
@@ -159,9 +156,8 @@ def capacity(self):
     '''
     RelatedJobsMixin
     '''
-    def _get_active_jobs(self):
-        return UnifiedJob.objects.filter(instance_group=self,
-                                         status__in=ACTIVE_STATES)
+    def _get_related_jobs(self):
+        return UnifiedJob.objects.filter(instance_group=self)
```
Comment (wwitzel3) [inline awx/main/models/ha.py:160]: Nice removing all of the extra filtering since the mixin already applies this filter.

**Review by wwitzel3 — APPROVED** [review]
Nice set of changes here.

**Pull request merged**